### PR TITLE
[ODS-6191] Security Vulnerability Check on Release Tags Added

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

**Comment by semalaiappan** [comment]
@simpat-jesus   Once PR is approved and will remove this conidtion   pull_request:
    branches: [main]  .   Finally after merge , will check nightly scheduler is running or not before closing this ticket .   Scope of this ticket is only ODS repo.

**Comment by semalaiappan** [comment]
image

**Comment by semalaiappan** [comment]
@stephenfuqua  @vimayya   https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Actions/pull/67 PR requires approval this ticket

File: .github/workflows/Security Vulnerability Check on Release Tags.yml
Comment (axelmarquezh) [inline .github/workflows/Security Vulnerability Check on Release Tags.yml:12]: Can we remove this PR trigger?

Consider adding a `workflow_dispatch`, it could come handy if we need to test the workflow, perhaps even include the tags as a parameter, although that could be too much work?

File: .github/workflows/Security Vulnerability Check on Release Tags.yml
Comment (axelmarquezh) [inline .github/workflows/Security Vulnerability Check on Release Tags.yml:53]: Could we remove this `Final-Check` job and instead implement the check as recommended [here](https://stackoverflow.com/a/70841204)?

File: .github/workflows/Security Vulnerability Check on Release Tags.yml
Comment (axelmarquezh) [inline .github/workflows/Security Vulnerability Check on Release Tags.yml:44]: Consider using Safire instead of HTML; I think it integrates seamlessly with GitHub actions, and we would see the results directly in the workflow results without needing to download the artifact.

Is my understanding correct @simpat-jesus?

**Review by axelmarquezh — CHANGES_REQUESTED** [review]

File: .github/workflows/Security Vulnerability Check on Release Tags.yml
Comment (semalaiappan) [reply .github/workflows/Security Vulnerability Check on Release Tags.yml:12]: @axelmarquezh  Please see this note https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/pull/951#issuecomment-1944175159 . I already mentioned here .  no need  of workflow_dispatch since it's nightly job requirement in this ticket

File: .github/workflows/Security Vulnerability Check on Release Tags.yml
Comment (axelmarquezh) [inline .github/workflows/Security Vulnerability Check on Release Tags.yml:23]: Can we temporarily add the tag `v5.3` just to see the pipeline fail? (It should detect a vulnerability in the _System.Data.SqlClient_ package).

**Review by axelmarquezh — CHANGES_REQUESTED** [review]

File: .github/workflows/Security Vulnerability Check on Release Tags.yml
Comment (semalaiappan) [reply .github/workflows/Security Vulnerability Check on Release Tags.yml]: https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/actions/runs/7984100004/job/21800366216?pr=951#step:3:19  Yes It's failing and able to detect it

Comment (semalaiappan) [reply .github/workflows/Security Vulnerability Check on Release Tags.yml]: image

**Comment by semalaiappan** [comment]
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/actions/runs/7984135324

File: .github/workflows/Security Vulnerability Check on Release Tags.yml
Comment (semalaiappan) [reply .github/workflows/Security Vulnerability Check on Release Tags.yml]: Done . Thanks

**Review by axelmarquezh — APPROVED** [review]

**Review by vimayya — APPROVED** [review]

**Pull request closed without merging**
